packages/Dashboards/src/App.js

```diff
@@ -109,6 +115,32 @@ function App(history) {
               <ReportEdit />
             </Route>
 
+            <Route exact path="/report/:reportid/generate">
+              <GenerateReport />
+            </Route>
+
+            <Route exact path="/report/generate/entityselection/step1">
```

Comment:
Lets change this up when we start tying in the backend so we don't have the step information in the url. If we really need to access via URL I think that #step1 would be more appropriate but this isn't a show stopper for me